support/cas-server-support-oidc-core-api/src/main/java/org/apereo/cas/oidc/web/OidcAccessTokenResponseGenerator.java

```diff
@@ -40,8 +41,17 @@ public OidcAccessTokenResponseGenerator(final IdTokenGeneratorService idTokenGen
         val accessToken = result.getGeneratedToken().getAccessToken();
         accessToken.ifPresent(token -> {
             val oidcRegisteredService = (OidcRegisteredService) result.getRegisteredService();
+            OAuth20JwtAccessTokenEncoder.builder()
```

Comment:
The move of OAuth20AccessTokenEncoder.builder() is to fix a null point exception in OidcImplicitIdTokenAuthorizationResponseBuilder (subclass of OAuth20TokenAuthorizationResponseBuilder) caused by the commit 76005b942b2f84689fa498cf8af8572e8d18a13c. This commit expects OAuth20JwtAccessTokenEncoder.encode() to be called before OidcIdTokenGeneratorService.generate().